examples/port-2-google-connection/main.tf

```diff
@@ -3,6 +3,36 @@ provider "equinix" {
   client_secret = var.equinix_client_secret
 }
 
+provider "google" {
+  region      = var.gcp_region
+  project     = var.gcp_project_id
+  zone        = var.gcp_zone
+  credentials = var.gcp_credentials_path
+}
+
+resource "google_compute_network" "port-gcp" {
+  project                 = var.gcp_project_id
+  name                    = var.gcp_network_name
+  mtu                     = var.gcp_network_mtu
+  auto_create_subnetworks = var.gcp_network_auto_create_subnetwork
+}
+
+resource "google_compute_router" "port-gcp" {
```

Comment:
Are the network and router just being created for the GCP interconnect so that we can get an auth key for the connection? There isn't a way to get the auth key for an ethernet type connection?

Totally fine if so; I just want to understand this more.